### PR TITLE
[#2707] Require svg images and adapt "multiple" i18n

### PR DESCRIPTION
**Pull request by tangrammer**
- [ ] **Update release notes if necessary**

no more nginx config needed just adding the required webpack & jsx logic to include and read the files

### CODE REVIEW

**Review by dlebrero — CHANGES_REQUESTED** [review]
The problem is that the images are not in the container:

```
Daniels-Mac-mini:akvo-authorization dlebrero$ for i in `kubectl get pods -l "app=lumen" -o jsonpath="{.items[*].metadata.name}"`; do kubectl exec -c lumen-client $i find . | grep type-text.svg ; done
Daniels-Mac-mini:akvo-authorization dlebrero$
```

The nginx config is not required as the "location / {" is already doing the same.

**Comment by tangrammer** [comment]
So adding another fix approach

**Review by marvinkome — APPROVED** [review]
Awesome

File: client/src/translations/en.json [hunk 1]
```diff
@@ -222,7 +222,7 @@
   "min_value": "Min: {val}",
   "min": "Min",
   "misc": "Other options",
-  "multiple": "Multiple",
+  "multiple": "Multiple values",
```
Comment (kardan) [inline client/src/translations/en.json:225]: I suggest we change the key to "multiple_values".

Comment (kardan) [reply client/src/translations/en.json:225]: Was meant as a suggestion, I could still merge the fix if we want it this Friday afternoon!

**Comment by tangrammer** [comment]
@kardan not so straightforward to change the i18n key thus we use the column type to point it, so to make this change it'll imply to create a new i18n key for each backend column key eg: multiple, multiple_type, text, text_type .... 🤔 so merging now the PR and we can think better on this improvement in a different issue

**Pull request merged**